### ISSUE TITLE
[Android] [com.facebook.katana] Package manager has died

### ISSUE THREAD

**Issue by fgarcia5** [ohh2ahh/AppAvailability#63]
When I try to check if the Facebook application is installed (com.facebook.katana) I get the following error, being that the App is installed:
![image](https://user-images.githubusercontent.com/11658144/80522454-fcd77380-8962-11ea-82fa-e4d7b2ac0c23.png)

This does not happen if I check another application, for example Instagram (com.instagram.android).

Device: Samsung J5 - Android 6.0
**Cordova-android:** 8.1.0
**Cordova plugins:**
- cordova-open-native-settings 1.5.2 "Native settings"
- cordova-plugin-androidx 1.0.2 "cordova-plugin-androidx"
- cordova-plugin-androidx-adapter 1.1.0 "cordova-plugin-androidx-adapter"
- cordova-plugin-app-version 0.1.9 "AppVersion"
- **cordova-plugin-appavailability 0.4.2 "AppAvailability"**
- cordova-plugin-background-mode 0.7.3 "BackgroundMode"
- cordova-plugin-badge 0.8.8 "Badge"
- cordova-plugin-device 2.0.3 "Device"
- cordova-plugin-dialogs 2.0.2 "Notification"
- cordova-plugin-firebasex 9.1.1 "Google Firebase Plugin"
- cordova-plugin-geolocation 4.0.2 "Geolocation"
- cordova-plugin-inappbrowser 3.2.0 "InAppBrowser"
- cordova-plugin-local-notification 0.9.3 "LocalNotification"
- cordova-plugin-network-information 2.0.2 "Network Information"
- cordova-plugin-splashscreen 5.0.3 "Splashscreen"
- cordova-plugin-statusbar 2.4.3 "StatusBar"
- cordova-plugin-vibration 3.1.1 "Vibration"
- cordova-plugin-whitelist 1.3.4 "Whitelist"
- cordova-plugin-x-socialsharing 5.6.5 "SocialSharing"
- es6-promise-plugin 4.2.2 "Promise"

Here my code:
`appAvailability.check(
            'com.facebook.katana',
            function(info) {
            },
            function(error) {
                console.log(error);
                //error get "Package manager has died"
            }
        );`

I have exhausted the alternatives I had at my disposal to solve it and I couldn't. 

Someone could help me? 

Thank you

**Comment by fgarcia5** [comment]
I have found a workaround for this error:
![image](https://user-images.githubusercontent.com/11658144/81213309-8fec5b00-8fac-11ea-8afd-cf078a2d8642.png)

Could you check it out and add to the repo

**Comment by rcasunshare** [comment]
> I have found a workaround for this error:
> ![image](https://user-images.githubusercontent.com/11658144/81213309-8fec5b00-8fac-11ea-8afd-cf078a2d8642.png)
> 
> Could you check it out and add to the repo

Do you have this plugin updated? because in Android it doesn't work with Facebook and in + iOS 11 nothing works directly.

**Comment by fgarcia5** [comment]
> > I have found a workaround for this error:
> > ![image](https://user-images.githubusercontent.com/11658144/81213309-8fec5b00-8fac-11ea-8afd-cf078a2d8642.png)
> > Could you check it out and add to the repo
> 
> Do you have this plugin updated? because in Android it doesn't work with Facebook and in + iOS 11 nothing works directly.

Yes, I have implemented it on both platforms and it works correctly.
If you want, give me more context and information about your project (if possible share the repository) and I will try to help you

**Comment by fdhdsjkfhds** [comment]
> > > I have found a workaround for this error:
> > > ![image](https://user-images.githubusercontent.com/11658144/81213309-8fec5b00-8fac-11ea-8afd-cf078a2d8642.png)
> > > Could you check it out and add to the repo
> > 
> > 
> > Do you have this plugin updated? because in Android it doesn't work with Facebook and in + iOS 11 nothing works directly.
> 
> Yes, I have implemented it on both platforms and it works correctly.
> If you want, give me more context and information about your project (if possible share the repository) and I will try to help you

Can you please share what modifications you made to allow for ios 11+ please, I'm stuck trying to find a solution for this.

**Comment by fgarcia5** [comment]
> > > > I have found a workaround for this error:
> > > > ![image](https://user-images.githubusercontent.com/11658144/81213309-8fec5b00-8fac-11ea-8afd-cf078a2d8642.png)
> > > > Could you check it out and add to the repo
> > > 
> > > 
> > > Do you have this plugin updated? because in Android it doesn't work with Facebook and in + iOS 11 nothing works directly.
> > 
> > 
> > Yes, I have implemented it on both platforms and it works correctly.
> > If you want, give me more context and information about your project (if possible share the repository) and I will try to help you
> 
> Can you please share what modifications you made to allow for ios 11+ please, I'm stuck trying to find a solution for this.

I can't help you without specifying the error.

Don't forget to add "LSApplicationQueriesSchemes", see:
https://github.com/ohh2ahh/AppAvailability#add-url-schemes-to-the-whitelist
